qemu/tests/fio_windows.py

```diff
@@ -0,0 +1,105 @@
+import re
+import time
+import logging
+
+from autotest.client.shared import error
+from autotest.client.shared import data_dir
+from autotest.client import utils
+
+from virttest import utils_misc
+from virttest import aexpect
+
+
+@error.context_aware
+def run(test, params, env):
+    """
+    KVM guest stop test:
+    1) Log into a guest
+    2) Check is fio.msi installed , download and
+       install it if not installed.
+    3) Start fio in guest
+    4) Check vm is alive
+    5) Stop fio process and clean temp file.
+
+    :param test: kvm test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+
+    def loop_session_cmd(session, cmd):
+        def session_cmd(session, cmd):
+            try:
+                return session.cmd_status(cmd, timeout=120) == 0
+            except (aexpect.ShellStatusError, aexpect.ShellTimeoutError):
+                pass
+
+        count = 0
+        while count < 3:
+            ret = session_cmd(session, cmd)
+            if ret is not None:
+                return ret
+            count += 1
+        return None
+
+    tmp_dir = data_dir.get_tmp_dir()
+    install_path = params["install_path"].rstrip("\\")
+    start_cmd = params.get("start_cmd")
+    stop_cmd = "taskkill /T /F /IM fio.exe"
+    stop_cmd = params.get("stop_cmd", stop_cmd)
+    check_running_cmd = "tasklist|findstr /I fio"
+    check_running_cmd = params.get("check_running_cmd", check_running_cmd)
+    test_installed_cmd = 'dir "%s"|findstr /I fio' % install_path
+    test_installed_cmd = params.get("check_installed_cmd", test_installed_cmd)
+
+    vm = env.get_vm(params["main_vm"])
+    vm.verify_alive()
+    timeout = float(params.get("login_timeout", 360))
+    session = vm.wait_for_login(timeout=timeout)
+    session_serial = vm.wait_for_serial_login(timeout=timeout)
+    try:
+        installed = session.cmd_status(test_installed_cmd) == 0
```

Comment:
+    test_installed_cmd = 'dir "%s"|findstr /I fio' % install_path

This cmd can just check whether there is fio file, can not check whether it's installed or not, right?